Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest-breadcrumbs/ContestBreadcrumbs.tsx

```diff
@@ -52,7 +52,8 @@ const ContestBreadcrumbs = () => {
                 dispatch(updateContestCategoryBreadcrumbItem({ elements: selectedCategoryBreadcrumbItems }));
             }
         }
-    }, [ contestCategories, breadcrumbItems.length, contestDetails, contestId, dispatch ]);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [ contestCategories, breadcrumbItems.length, contestDetails, dispatch ]);
```

Comment:
Explain in a comment what is the problematic dependency/ies and why.

Reply:
done